examples/cli/indexers/1-evm.indexer.ts

```diff
@@ -1,3 +1,5 @@
+import { db } from "@/lib/db";
+import { ethereumUsdcTransfers } from "@/lib/schema";
```

Comment:
Can you move local imports at the bottom? I like to always have global imports at the top, followed by local import (with alias), followed by local import with relative path.